lists/my.csv

```diff
@@ -22,6 +22,8 @@ http://break-a-leg.blogspot.com/,CULTR,Culture,2016-06-10,OONI,
 https://www.breastcancerfoundation.org.my/,PUBH,Public Health,2018-12-11,sinarproject,
 http://brimas.www1.50megs.com/,ENV,Environment,2014-04-15,citizenlab,
 http://cartoonistsrights.org/designeractivist-fahmy-reza-on-trial-for-online-dissent-in-malaysia/,POLR,Political Criticism,2016-06-10,OONI,
+https://www.cato.org/blog/free-download-banned-malay-edition-islam-without-extremes,REL,Religion,2019-05-06,sinarproject,Post on free download of book "Islam Without Extremes" banned by Home Ministry
+https://object.cato.org/sites/cato.org/files/wp-content/uploads/islam_tanpa_keekstreman_-_2019_revised.pdf,REL,Religion,2019-05-06,sinarproject,Link to free download of book on "Islam Without Extremes" banned by Home Ministry
```

Comment:
When accessed from a browser, this seems to redirect to: https://www.cato.org/sites/cato.org/files/wp-content/uploads/islam_tanpa_keekstreman_-_2019_revised.pdf 

Perhaps include this URL?